source/developers-guide/code-snippets/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Code snippet collection
+github_link: developers-guide/code-snippets/index.md
+shopware_version: 5.2.0
+indexed: true
+---
+
+This article will provide some useful code snippets that can help you when developing Shopware plugins. Please feel free to add additional snippets via GitHub!
+
+## Read  config from a different shop than the current one
+
+```php
+/** @var \Shopware_Components_Config $config */
+$config = \Shopware()->Container()->get('config');
+$config->setShop($shop);
```

Comment:
@mitelg Okay this example obviously sucks and I will add a new example to the PR.

@dnoegel I think there is a need for a `ConfigService` that offers a `getByContext()` method.  Or a `get(ShopContext $context = null)` method that uses the current `ShopContext` by default. If you need a real life example where we would use the new service please ask me!